config/recipes/elastic-agent/multi-output.yaml

```diff
@@ -27,24 +27,20 @@ spec:
             path: /var/log
           name: varlog
   config:
-    id: 2d70a6f0-33a5-11eb-bb2f-418d0388a8cf
-    revision: 2
     agent:
       monitoring:
         enabled: true
         use_output: monitoring
         logs: true
         metrics: true
     inputs:
-    - id: 2e187fb0-33a5-11eb-bb2f-418d0388a8cf
```

Comment:
I would keep this but try to make it match what Fleet would generate. The Fleet UI generates unique IDs per input, and we should try to stay consistent between standalone and fleet policies. 

A policy generated in the fleet UI would name this something like `system/logfile-2e187fb0-33a5-11eb-bb2f-418d0388a8cf`. The PR that added unique IDs is https://github.com/elastic/kibana/pull/127343

The logfile input should work without it but there is an effort to transparently convert logfile to filestream under the hood and in that case we'll want the input and stream IDs to be unique.